### ISSUE TITLE
Issue with "aria-selected": cannot find the source of "false" as the implicit default for tab

### ISSUE THREAD

**Issue by ianthedev** [mdn/content#14699]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected

#### What information was incorrect, unhelpful, or incomplete?
The source of "Implicit default for [`tab`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role)" cannot be found.

#### Specific section or headline?
[Values](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected#values)

#### What did you expect to see?
The specification saying `false` is the implicit default for `tab`.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/attributes/aria-selected`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/attributes/aria-selected/index.md
* Last commit: https://github.com/mdn/content/commit/54fd6eaad3924076e0b546e7eff1f6f466f6139f
* Document last modified: 2022-01-24T00:41:11.000Z

</details>

**Comment by thedivloop** [comment]
@ianthedev the sentence "The default value for aria-selected on a tab role is false." is already in the specification under tab.
Is it just the "implicit" word missing or is it something else?

**Comment by ianthedev** [comment]
@thedivloop Thanks. Could you please let me know where that sentence is? I could not find it in [Accessible Rich Internet Applications (WAI-ARIA) 1.3](https://w3c.github.io/aria/#tab) and [WAI-ARIA Authoring Practices 1.2](https://www.w3.org/TR/wai-aria-practices-1.2/#tabpanel).

**Comment by thedivloop** [comment]
> @thedivloop The problem is that [Accessible Rich Internet Applications (WAI-ARIA) 1.3](https://w3c.github.io/aria/#tab) and [WAI-ARIA Authoring Practices 1.2](https://www.w3.org/TR/wai-aria-practices-1.2/#tabpanel) do not say so, as far as I can see.

OK I think I understood your point. On MDN it is stated that the tab role default is false yet on the specs that you shared that point seems missing.

Under tab role (link you shared), I found this:
<img width="827" alt="Screen Shot 2022-04-09 at 23 02 04" src="https://user-images.githubusercontent.com/80190976/162581922-e8170021-6b2b-4564-8c99-5ec240d017da.png">

Does it answer your question?

**Comment by ianthedev** [comment]
Thank you very much! That answered my question.
